### PR TITLE
chore(js): inbox localization keys fix

### PR DESCRIPTION
**Pull request by LetItRock**
### What changed? Why was the change needed?
Inbox fixed the localization keys and moved some hardcoded texts.

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/js/src/ui/components/Notification/DefaultNotification.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
             'nt-text-foreground-alpha-400 nt-ml-6 nt-shrink-0 nt-float-right group-hover:nt-hidden'
           )}
         >
-          {formatToRelativeTime({ fromDate: new Date(props.notification.createdAt), locale })}
+          {formatToRelativeTime({ fromDate: new Date(props.notification.createdAt), locale: locale() })}
```
Comment (LetItRock) [inline packages/js/src/ui/components/Notification/DefaultNotification.tsx]: the context should pass the locale as the accessor

File: packages/js/src/ui/components/elements/Header/MoreActionsOptions.tsx [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
   const { archiveAll } = useArchiveAll();
   const { archiveAllRead } = useArchiveAllRead();
 
   return (
     <>
-      <ActionsItem label="Mark all as read" onClick={() => readAll({ tags: filter().tags })} icon={ReadAll} />
-      <ActionsItem label="Archive all" onClick={() => archiveAll({ tags: filter().tags })} icon={Archive} />
-      <ActionsItem label="Archive read" onClick={() => archiveAllRead({ tags: filter().tags })} icon={ArchiveRead} />
+      <ActionsItem
+        label={t('notifications.actions.readAll')}
```
Comment (LetItRock) [inline packages/js/src/ui/components/elements/Header/MoreActionsOptions.tsx:18]: moved the hardcoded values to localization keys

File: packages/js/src/ui/config/defaultLocalization.ts [hunk 3]
```diff
@@ -1,17 +1,20 @@
+// TODO check all localization keys
 export const defaultLocalization = {
   locale: 'en-US',
-  'inbox.title': 'Inbox',
```
Comment (LetItRock) [inline packages/js/src/ui/config/defaultLocalization.ts:3]: not used

File: packages/js/src/ui/context/LocalizationContext.tsx [hunk 4]
```diff
@@ -25,11 +25,13 @@ export const LocalizationProvider = (props: LocalizationProviderProps) => {
     return value;
   };
 
+  const locale = createMemo(() => localization().locale);
```
Comment (LetItRock) [inline packages/js/src/ui/context/LocalizationContext.tsx:36]: when the localization prop changes we want to update the `locale` as well

**Review by SokratisVidros — APPROVED** [review]

**Review by BiswaViraj — APPROVED** [review]

**Review by desiprisg — APPROVED** [review]

File: packages/js/src/ui/config/defaultLocalization.ts (hunk 3 above)
Comment (desiprisg) [reply packages/js/src/ui/config/defaultLocalization.ts:3]: This was probably the first one we added as an example!

**Pull request merged**